src/types/state.ts

```diff
@@ -131,6 +131,17 @@ export const UUID = ShortString;
 /** ***************
  * Current state *
  *****************/
+
+const UpdateStatus = t.union([
+	t.literal('rejected'),
+	t.literal('downloading'),
+	t.literal('downloaded'),
+	t.literal('applying changes'),
```

Comment:
Possible to simplify this string to `applying`?

Reply:
This is what the server expects. We try to keep messaging consistent between api/sdk/dashboard